### PR TITLE
Feature/bc-update-hdc-api-v2

### PR DESCRIPTION
**Pull request by Ben-Carpenter**

### CODE REVIEW

File: src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs [hunk 1]
```diff
@@ -28,10 +28,19 @@ public enum SubmissionType
...
+        }
 
         [JsonProperty("commitStatus")]
         public string CommitStatus { get; set; }
+        
+        [JsonProperty("isExtensionInf")]
+        public bool IsExtensionInf { get; set; }
+
+        [JsonProperty("isUniversal")]
+        public bool isUniversal { get; set; }
```
Comment (gt-downunder) [inline src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs:40]: Should be "IsUniversal"

File: src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs [hunk 2]
```diff
@@ -28,10 +28,19 @@ public enum SubmissionType
...
+
+        [JsonProperty("isDeclarativeInf")]
+        public bool isDeclarativeInf { get; set; }
```
Comment (gt-downunder) [inline src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs:43]: Should be "IsDeclarativeInf"

Comment (Ben-Carpenter) [reply src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs:43]: yes, thank you for catching that.

File: src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs (hunk 1 above)
Comment (Ben-Carpenter) [reply src/Microsoft.Devices.HardwareDevCenterManager/Models/Submission.cs:40]: yes, thank you for catching that.

**Pull request merged**